Curry.xcodeproj/project.pbxproj

```diff
@@ -454,7 +454,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 4.0;
```

Comment:
Does this mean we should do a major version bump? @sharplet I feel like you know more about this compatibility than I do.

Reply:
Yeah I'm pretty sure this would break compilation on Xcode 8.x

Reply:
😞 _sigh_ that's what I was afraid of. Curry 4.0 here we come!